### PR TITLE
TCD-2903: Backtrace - stadia and cryengine

### PR DESCRIPTION
**Pull request by LoryApiFortress**
### Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### CODE REVIEW

File: docs/error-reporting/platform-integrations/cry-engine.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+id: cry-engine
+title: CryEngine Integration Guide
+sidebar_label: CryEngine
+description: Integrate your CryEngine game into Backtrace for crash reporting.
```
Comment (LoryApiFortress) [inline docs/error-reporting/platform-integrations/cry-engine.md:5]:
```suggestion
description: Integrate your CryEngine game into Backtrace for crash reporting.
```

File: docs/error-reporting/platform-integrations/cry-engine.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+Backtrace provides first-class support for the native crash reporting format of CryEngine. The basic integration consists of modifying a configuration file to configure your game to use Backtrace as the crash reporting server. The system accepts crash reports from the game client, automatically extracts and indexes all the metadata provided by the client, and accepts any file attachments.
+
+## Supported Platforms
+
+Backtrace's integration with CryEngine supports games deployed on Windows. If you are deploying to game consoles (PS4, Xbox One), use [Backtrace's Game Console Integrations](https://support.backtrace.io/hc/en-us/sections/360007642051-Video-Game-Technologies).
+
+For On-Premise (self-hosted) users: CryEngine crash ingestion requires specific packages. Contact support for detailed instructions and information.
+
+## How to Integrate
```
Comment (LoryApiFortress) [inline docs/error-reporting/platform-integrations/cry-engine.md:22]:
```suggestion
## How To Integrate
```

File: docs/error-reporting/platform-integrations/cry-engine.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
+---
+id: cry-engine
```
Comment (lysannep) [inline docs/error-reporting/platform-integrations/cry-engine.md:2]:
```suggestion
id: cryengine
```

File: sidebars.js [hunk 4]
```diff
@@ -434,6 +434,8 @@ module.exports = {
                         'error-reporting/platform-integrations/apache',
                         'error-reporting/platform-integrations/visual-studio',
                         'error-reporting/platform-integrations/plcrash-reporter',
+                        'error-reporting/platform-integrations/stadia',
+                        'error-reporting/platform-integrations/cry-engine',
```
Comment (lysannep) [inline sidebars.js:438]:
```suggestion
                        'error-reporting/platform-integrations/cryengine',
```

**Review by lysannep — APPROVED** [review]

File: docs/error-reporting/platform-integrations/cry-engine.md
Comment (lysannep) [inline docs/error-reporting/platform-integrations/cry-engine.md:1]: Can you also rename the file from `cry-engine.md` to `cryengine.md`?

**Pull request merged**